sormas-ui/src/main/java/de/symeda/sormas/ui/task/TaskList.java

```diff
@@ -87,7 +87,7 @@ protected void drawDisplayedEntries() {
 			if (UserProvider.getCurrent().hasUserRight(UserRight.TASK_EDIT)) {
 				listEntry.addEditListener(
 					i,
-					(ClickListener) event -> ControllerProvider.getTaskController().edit(listEntry.getTask(), TaskList.this::reload));
+					(ClickListener) event -> ControllerProvider.getTaskController().edit(listEntry.getTask(), TaskList.this::reload, true));
```

Comment:
TaskList is also used in contacts and events, so this can't be generically "true"